velox/exec/tests/OrderByTest.cpp

```diff
@@ -463,6 +468,9 @@ TEST_F(OrderByTest, spill) {
   EXPECT_LT(0, stats[0].operatorStats[1].spilledBytes);
   EXPECT_EQ(1, stats[0].operatorStats[1].spilledPartitions);
   EXPECT_EQ(2, stats[0].operatorStats[1].spilledFiles);
+  EXPECT_GT(
+      toPlanStats(task->taskStats()).at(orderById).cpuWallTiming.wallNanos,
```

Comment:
Use a temp variable to capture the result of toPlanStats(task->taskStats()) to avoid computing it twice.

Reply:
Updated.